extensions.html

```diff
@@ -0,0 +1,332 @@
+          //      company: "Your Company", companyURL: "http://example.com/" },
+          //],
+
+          // name of the WG
+          wg:           "Web Platform Working Group",
+
+          // URI of the public WG page
+          wgURI:        "http://www.w3.org/WebPlatform/WG/",
+          license:	"w3c-software-doc",
+
+          // name (with the @w3c.org) of the public mailing to which comments are due
+          wgPublicList: "public-webapps",
+
+          // URI of the patent status for this WG, for Rec-track documents
+          // !!!! IMPORTANT !!!!
+          // This is important for Rec-track documents, do not copy a patent URI from a random
+          // document unless you know what you're doing. If in doubt ask your friendly neighbourhood
+          // Team Contact.
+          wgPatentURI:  "http://www.w3.org/2004/01/pp-impl/83482/status",
+      };
+    </script>
+
+    <style type="text/css">
+      .event {
+        font-family: monospace;
+        color: #459900;
+      }
+
+      pre.idl {
+        white-space: pre-wrap;
+      }
+    </style>
+  </head>
+  <body>
+    <section id='abstract'>
+      Extensions to the base Gamepad specification to enable a access to more
+      advanced device capabilities.
+    </section>
+
+    <section id='sotd'>
+      If you have comments for this spec, please send them to
+      <a href="mailto:public-webapps@w3.org">public-webapps@w3.org</a>
+      with a Subject: prefix of <code>[gamepad]</code>. See
+      <a href="https://www.w3.org/Bugs/Public/buglist.cgi?product=WebAppsWG&component=Gamepad&resolution=---">Bugzilla</a>
+      for this specification's open bugs.
+    </section>
+
+    <section id='introduction' class='informative'>
+
+      <h2>Introduction</h2>
+
+      <p>The Gamepad API provides a tightly scoped interface to gamepad devices
+      that's focused on the most common elements of those devices. Namely: Axis
+      and button input. It specifically excludes support for more complex
+      devices that may also such as those that that do motion sensing or
+      haptic feedback.</p>
+
+      <p>However some uses of gamepads, such as those paired with Virtual
+      Reality headsets, rely heavily on those more advanced features. This
+      supplemetary spec describes extensions to the base API to accomodate those
+      use cases. If they prove to be broadly useful the hope is that they will
+      be eventually merged into the main spec.</p>
+
+    </section>
+
+    <section>
+      <h2><dfn>GamepadHand</dfn> Enum</h2>
+      <p>
+        This enum defines the set of possible hands a gamepad may be held by.
+      </p>
+
+      <pre class="idl">
+        enum GamepadHand {
+            "",
+            "left",
+            "right"
+        };
+      </pre>
+
+      <dl data-dfn-for="GamepadHand">
+        <dt></dt>
+        <dd>
+          The empty string indicates the hand holding the gamepad is unknown or
```

Comment:
personally, I think this would read better: `the hand holding` should be `the hand that is holding`